eZ/Bundle/EzPublishElasticsearchSearchEngineBundle/EzPublishElasticsearchSearchEngineBundle.php

```diff
@@ -20,7 +21,7 @@
 use eZ\Publish\Core\Base\Container\Compiler\Search\FieldRegistryPass;
 use eZ\Publish\Core\Base\Container\Compiler\Search\SignalSlotPass;
 
-class EzPublishElasticsearchBundle extends Bundle
+class EzPublishElasticsearchSearchEngineBundle extends Bundle
```

Comment:
Elastic Search has never been exposed in a stable release yet, so while we can mention it, it should be maybe in a separate category of notable information. Like change from 2015.01 (6.0.0-alpha1)